testsuite/cli/src/client_proxy.rs

```diff
@@ -514,40 +514,68 @@ impl ClientProxy {
             space_delim_strings[0]
         );
         ensure!(
-            space_delim_strings.len() == 2,
+            space_delim_strings.len() == 3,
             "Invalid number of arguments for removing validator"
         );
         let (account_address, _) =
             self.get_account_address_from_parameter(space_delim_strings[1])?;
-        match self.faucet_account {
-            Some(_) => self.association_transaction_with_local_faucet_account(
-                transaction_builder::encode_remove_validator_script(account_address),
-                is_blocking,
-            ),
-            None => unimplemented!(),
+        let private_key = Ed25519PrivateKey::from_encoded_string(space_delim_strings[2])?;
```

Comment:
Why are we changing this? It seems like the association should still have this ability? Although arguably -- at some point, we will have to figure out how we balance out the real governance from a test governance (which this is still, right?)

Reply:
With this PR: the validator operator may add/remove its validator from the set. The Association may certify/decertify a validator. It's not a change in governance - just change in naming.

With that, it was simpler to change these tests to call add/remove validator by the validator's operator (previously - those were called by the Association). Because if I change those tests to call the certify/decertify by the Association, then to become active in the set again after the Association called "certify", the validator's operator needs to additionally call "LibraSystem::add_validator". So the test will become more complicated - but I can change it.

Are you saying, these new APIs are not how we envision governance to work?